README.md

```diff
@@ -122,37 +159,109 @@ For more code samples, take a look at Keen's `docs <https://keen.io/docs/api/?py
         "actor_property": "user.email"
     }
     keen.funnel([step1, step2], timeframe="today") # => [2039, 201]
-
+```
 
 To return the full API response from a funnel analysis (as opposed to the singular "result" key), set `all_keys=True`.
 
 For example, `keen.funnel([step1, step2], all_keys=True)` would return "result", "actors" and "steps" keys.
 
-Delete Events
-`````````````
+#### Saved queries
+
+You can manage your saved queries from the Keen Python client, here's an example:
+
+```python
+    # Create a saved query
+    saved_query_attributes = {
+    "refresh_rate": 0,
+    "query": {
+      "analysis_type": "count",
+      "event_collection": "purchases",
+      "timeframe": "this_2_weeks",
+      "filters": [
+      {
+        "property_name": "price",
+        "operator": "gte",
+        "property_value": 1.00
+      }]},
+      "metadata": {
+        "display_name": "Purchases (past 2 weeks)",
+      }
+    }
+    
+    keen.saved_queries.create("saved-query-slug", saved_query_attributes)
+    
+    # Get all saved queries
+    keen.saved_queries.all()
+
+    # Get one saved query
+    keen.saved_queries.get("saved-query-slug")
+
+    # Get saved query with results
+    keen.saved_queries.results("saved-query-slug")
+
+    # Update a saved query
+    # You can auto cache a saved query with refresh_rate, see https://keen.io/docs/api/#auto-cache-a-saved-query for more info
+    saved_query_attributes = { refresh_rate: 14400 } 
+    keen.saved_queries.update("saved-query-slug", saved_query_attributes)
+
+    # Delete a saved query
+    keen.saved_queries.delete("saved-query-slug")
+```
 
-The Keen IO API allows you to `delete events <https://keen.io/docs/api/#delete-events>`_ from event collections, optionally supplying filters, timeframe or timezone to narrow the scope of what you would like to delete.
+### Advanced Usage
 
-You'll need to set your master_key.
+#### Overwriting event timestamps
 
-.. code-block:: python
+Two time-related properties are included in your event automatically. The properties `keen.timestamp`
+and `keen.created_at` are set at the time your event is recorded. You have the ability to overwrite the
+`keen.timestamp` property. This could be useful, for example, if you are backfilling historical data. Be
+sure to use [ISO-8601 Format](https://keen.io/docs/event-data-modeling/event-data-intro/#iso-8601-format).
 
+Keen stores all date and time information in UTC!
```

Comment:
I propose slightly changing this section by providing a code snippet to update more than a single event. 
_Backfilling historical data_ or _Overwriting event timestamps_  is more likely done for several events than just one.

Another option is to provide a separate code snippet with  `keen.add_events` to demonstrate update of more than a single event. Example of what I mean is [here](https://github.com/krzychb/esp32-everest-run/blob/master/log-book.md#wi-fi-signal-coverage)

I was looking for such option since couple of days and have finally figured this out only thanks to help by adam (at) keen-io. 

Thank you Adam!

P.S. To be precise who I mean, I see only one Adam on https://keen.io/team/ :smile: